controllers/kyma_controller.go

```diff
@@ -229,23 +224,30 @@ func (r *KymaReconciler) HandleInitialState(ctx context.Context, kyma *v1beta1.K
 func (r *KymaReconciler) HandleProcessingState(ctx context.Context, kyma *v1beta1.Kyma) error {
```

Comment:
Btw. this controller contains a bunch of methods that are exported, but since they don't belong to any interface implementation or are not used from other packages they can be made package-private. Can you check them please?

Reply:
"Find Usage" Tool shows no use outside the KymaReconciler and I know or can think of no other piece of code that might use these function.
Also making them package-private breaks no test and manual testing works.